cmd/kube-proxy/app/server.go

```diff
@@ -551,8 +552,7 @@ func newProxyServer(config *kubeproxyconfig.KubeProxyConfiguration, master strin
 		return nil, err
 	}
 
-	s.NodeIP = detectNodeIP(s.Client, s.Hostname, config.BindAddress)
-	klog.InfoS("Detected node IP", "address", s.NodeIP.String())
+	s.PrimaryIPFamily, s.NodeIPs = detectNodeIPs(s.Client, s.Hostname, config.BindAddress)
```

Comment:
is not better to keep the log here and log the primary ip family too?

Reply:
I had moved it so that it only would only log about "detecting" the node IP if it actually _detected_ it (rather than you specifying `--bind-address`).

Primary IP family is really irrelevant unless it falls back to dual-stack, in which case it will be logged in that context.

Reply:
I think that in this push you completely drop the logging